{
  "@type": "NewsArticle",
  "identifier": "2022--04--11--en--myfeed--HackerNews--NewsArticle--30986732",
  "url": "https://news.ycombinator.com/item?id=30986732",
  "headline": "Atlassian successfully restores just over 1 out of 3 affected customers data",
  "publisher": {
    "@type": "Organization",
    "name": "HackerNews",
    "url": "https://news.ycombinator.com",
    "logo": "https://hn.buzzing.cc/avatar.png"
  },
  "description": "",
  "keywords": [],
  "author": {
    "@type": "Person",
    "name": "taspeotis",
    "url": "https://news.ycombinator.com/user?id=taspeotis"
  },
  "discussionUrl": "https://news.ycombinator.com/item?id=30986732",
  "sameAs": "https://confluence.status.atlassian.com/incidents/hf1xxft08nj5",
  "dateCreated": "2022-04-11T10:35:51.708Z",
  "datePublished": "2022-04-11T10:13:08.000Z",
  "dateModified": "2022-04-11T10:35:51.708Z",
  "interactionStatistic": [
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "LikeAction"
      },
      "userInteractionCount": 10
    },
    {
      "@type": "InteractionCounter",
      "interactionType": {
        "@type": "CommentAction"
      },
      "userInteractionCount": 5
    }
  ],
  "headline_zh-Hans": "阿特拉斯公司成功恢复了3个受影响客户中略多于1个的数据",
  "headline_zh-Hant": "阿特拉斯公司成功恢復了3個受影響客戶中略多於1個的數據",
  "@context": [
    "https://schema.org",
    {
      "@vocab": "http://schema.org/",
      "@language": "en",
      "headline_zh-Hans": {
        "@id": "headline",
        "@language": "zh-Hans"
      },
      "headline_zh-Hant": {
        "@id": "headline",
        "@language": "zh-Hant"
      },
      "@version": 1.1
    }
  ]
}